src/backend/backend.did

```diff
@@ -79,17 +79,30 @@ type ListUsersResponse = record {
   users : vec OisyUser;
   matches_max_length : nat64;
```

Comment:
This is the updated list of migration progress states, including failure reasons.